ros_cross_compile/data_collector.py

```diff
@@ -62,12 +71,14 @@ def timer(self, name: str):
             time_metric = Datum('{}-time'.format(name), elapsed,
                                 Units.Seconds.value, time.monotonic(), complete)
             self.add_datum(time_metric)
+            self.print_data(time_metric.unit, time_metric)
```

Comment:
I don't think we need to print every time a metric is created - i would rather print all at the end. That way it's not dispersed in between other output.